docs/troubleshoot/_category_.json

```diff
@@ -0,0 +1,8 @@
+{
+  "label": "Support and troubleshooting",
+  "position": 6,
+  "link": {
+    "type": "generated-index",
+    "description": "Support and troubleshooting"
```

Comment:
can we shorten title to one word?

Reply:
referenced from https://learn.microsoft.com/en-us/azure/aks/aks-support-help, i m wondering if there is a sidebar title guidance. lower case easier to read maybe?

Reply:
yeah I'm not familiar with a standard convention. Just looking at our current model. The "What is Ratify" doc also follows the same convention of capitalizing nouns. Either way we should probably standardize